### DISCUSSION TITLE
Welcome to hass-unfoldedcircle Discussions!

### DISCUSSION THREAD

**Discussion by JackJPowell** [JackJPowell/hass-unfoldedcircle#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ryanmsteed** [comment]
Hey @JackJPowell this is an awesome integration! my family uses it every day and loves it. Thanks so much for maintaining it.

One question I had - where may I update the IP of the remote? I had to change my home's network configuration and am unable to get your integration to connect again (I was able to get UCR's HA integration working again). Adding a new device seems to fail. I'm afraid to delete the integration and start over. Thanks!
